ThirdPartyNotice.txt

```diff
@@ -38,6 +38,7 @@ jiacfan/keychain, Copyright (c) 2015 Keybase
 mattn/go-ieproxy, Copyright (c) 2014 mattn
 mattn/go-ieproxy, Copyright (c) 2017 oliverpool
 mattn/go-ieproxy, Copyright (c) 2019 Adele Reed
+cpuguy83/go-md2man, Copyright (c) 2014 Brian Goff

```

Comment:
Out of curiosity, are these dependencies of cobra/doc?

Reply:
Yes indeed